src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -73,8 +73,8 @@
   "item.minecolonies.scepterlumberjack": "Forest Tool",
   "item.minecolonies.scepterlumberjack.usedend": "Position A set. Select Position B with a right-click or left-click the same position to finish.",
   "item.minecolonies.scepterlumberjack.usedstart": "Position B set. Select Position A with a left-click or right-click the same position to finish.",
-  "item.minecolonies.scepterlumberjack.restrictiontoobig": "An area of %d blocks is too large. The maximum restricted area for the Forester is %d blocks.",
```

Comment:
Maybe we call it box or so? I think volume might confuse some players

Reply:
A "box" could still be 2D though.

Reply:
Or "zone", since the setting to turn it on is called "zoning mode"

Reply:
That's also usually a 2D concept.

Reply:
area is probably still the best for 3d, btw your changed version now has "An volume"

Reply:
I apparently wasn't awake enough yet 🤦
Area sounds more like 2D than 3D in my opinion

Reply:
I've tweaked the wording a bit to use a mixture of "zone", "3D box", and "volume" where each seems most appropriate.

Reply:
Sorry, cos I am confused now reading through this - do we now need to click one corner higher than the other? Cos it didn't matter if we clicked the same Y level before, it did the area from bedrock to sky limit, but now the wording makes it sound like we need to click (for example) y64 one corner, and y74 other corner to get the height in?

Reply:
The actual behaviour has not been changed by this PR.  I originally thought as you did that it was a 2D region only (bedrock to sky) but it turns out that this is not the actual implementation.  The first version of this changed it back to a 2D region, but Ray said that being 3D was intended, so now it just updates the messages to make that clearer.

So yes, you need to click at different Y levels to get them to register a 3D box properly.  Having said that, there is a bit of a fudge built in, and it's only the base of the tree that has to be within the box, so the previous behaviour of clicking twice at the same ground Y still works to select an area if you're clicking on the same ground level that all of your trees are planted directly on.  But this will not find any trees planted 1y above the ground that you clicked on, which is how I noticed this issue in the first place.

i.e. if you click on a ground block at Y=63 twice, this will only accept the tree base (bottom log) being at Y=62-64 and will not accept Y=65 (i.e. ground, another block, then log).  This might be the source of some of the "my forester doesn't find trees if I put them up in planters or on hilly terrain" issues.

Reply:
I'm gonna have to test it. Cos unless I am misremembering, I am sure I set a zone for my forester in an underground colony. Forester kept dying above ground - I had to remove all the trees above ground between the x and z coordinates so that they would only chop trees where I planted them. Was doing an underground Hive style colony, so it was either in 1.18 or 1.16.  Unless things changed since then?  Will try and test it anyway.